subsys/bluetooth/controller/ll_sw/ull_adv_internal.h

```diff
@@ -163,6 +163,10 @@ void ull_adv_sync_update(struct ll_adv_sync_set *sync, uint32_t slot_plus_us,
 /* helper function to schedule a mayfly to get sync offset */
 void ull_adv_sync_offset_get(struct ll_adv_set *adv);
 
+/* helper function to reserve ACAD field in PDU buffer */
+uint8_t ull_adv_sync_acad_enable(struct lll_adv_sync *lll_sync,
+				 uint8_t acad_len, void **acad);
```

Comment:
I'm curious as to why the `acad` field here is a `void **` rather than `uint8_t **`. Can you elaborate?

Reply:
Isnt this outdated code.

`void *` is guaranteed by compilers to be assigned to pointers of other data types without explicit typecast (similar to why `malloc` returns `void *`).

Reply:
It is. I was reviewing commit-by-commit. You can disregard this.